spec/features/admin/admin_modifies_items_spec.rb

```diff
@@ -39,7 +39,7 @@
     expect(current_path).to eq(edit_admin_item_path(@item))
   end
 
-  xscenario 'can modify item details' do
+  scenario 'can modify item details' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.